arm/Microsoft.Compute/galleries/readme.md

```diff
@@ -1,13 +1,15 @@
-# Shared Image Gallery `[Microsoft.Compute/galleries]`
+# ComputeGalleries `[Microsoft.Compute/galleries]`
 
 This module deploys Share Image Gallery, with resource lock.
 
-## Resource types
+## Resource Types
 
 | Resource Type | Api Version |
 | :-- | :-- |
 | `Microsoft.Authorization/locks` | 2016-09-01 |
 | `Microsoft.Compute/galleries` | 2020-09-30 |
+| `Microsoft.Compute/galleries/images` | 2020-09-30 |
+| `Microsoft.Compute/galleries/images/providers/roleAssignments` | 2021-04-01-preview |
```

Comment:
RBAC does not seem up to date. Please check out the merge conflicts and re-use the rbac format Marius updated in main